pkg/ns/README.md

```diff
@@ -1,41 +1,5 @@
-### Namespaces, Threads, and Go
```

Comment:
Can you keep this file? It's internal development information.

Reply:
I've removed any files that are no longer in the website as of PR https://github.com/containernetworking/cni.dev/pull/43.
(That is, files in `pkg/` and in `vendor/`)